coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
```

Comment:
@Siddhartha1234 No. I only mean that in Python 3 `super()` doesn't need any arguments for that default use case :)